src/main/java/com/esri/samples/portal/oauth/README.md

```diff
@@ -1,42 +1,41 @@
 
 <h2>How it works</h2>
 
 <ol>
-    <li>Setup an <code>OAuthConfiguration</code> with the settings of an application registered in the ArcGIS platform.</li>
-    <li>Setup an <code>AuthenticationChallengeHandler</code> that challenges the user for authentication. You could
-    create a custom handler similar to the one created in this sample: <code>OAuthChallengeHandler</code>.</li>
-    <li>On trying to access a secured resource, the authentication challenge in invoked.</li>
-    <li><code>OAuthChallengeHandler</code> directs the user to a sign-in page (using a <code>WebView</code>) from ArcGIS platform.</li>
-    <li>On successful sign-in, the ArcGIS platform provides an authorization code.</li>
-    <li>Use the authorization code to create a <code>OAuthTokenCredentialRequest</code>. This will be used by the Runtime
-    to request an access token. The access token is then used to request a secured resource in the ArcGIS platform.</li>
+   <li> Set the `AuthenticationManager`'s `AuthenticationChallengeHandler` to the `DefaultAuthenticationChallengeHandler`.</li> 
+    <li> Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URL.</li> 
+    <li> Add the OAuth configuration to the authentication manager.</li> 
+    <li> Load a map with premium content requiring authentication to automatically invoke the default authentication handler.</li> 
 </ol>
 
 <h2>Relevant API</h2>
 
 <ul>
-    <li>AuthenticationChallengeHandler</li>
-    <li>OAuthConfiguration</li>
-    <li>OAuthTokenCredential</li>
-    <li>OAuthTokenCredentialRequest</li>
-    <li>Portal</li>
-    <li>PortalUser</li>
+  <li>AuthenticationManager</li>
+  <li>AuthenticationChallengeHandler</li>
+  <li>OAuthConfiguration</li>
+  <li>PortalItem</li>
 </ul>
 
 <h2>Additional information</h2>
 
-<p>The JavaFX <code>WebEngine</code> used in the <code>OAuthChallengeHandler</code> in this sample may not support rendering of some modern web elements returned by the <code>AuthorizationURL</code>. For this reason, we append <code>&display=classic</code> to the authorization URL, to ensure it renders properly.</p>
\ No newline at end of file
+<p>The workflow presented in this sample works for all SAML based enterprise (IWA, PKI, Okta, etc.) &amp; social (facebook, google, etc.) identify providers for ArcGIS Online or Portal. More information can be found <a href="https://doc.arcgis.com/en/arcgis-online/administer/enterprise-logins.htm">here</a>.</p>
+
+<p>For additional information on using Oauth in your app, see the <a href="https://developers.arcgis.com/documentation/core-concepts/security-and-authentication/mobile-and-native-user-logins/">Mobile and Native Named User Login</a> topic in our guide.</p>
```

Comment:
Oauth -> OAuth

I think we should also maintain and highlight the link to the wiki on our own Java pages. I find this very helpful and is a direct step by step guide for generating a Client ID. That info is kinda buried in the current link. Something like:

<p>"To setup and access your own app using this sample (using your own ArcGIS portal), see the <a href="https://github.com/Esri/arcgis-runtime-samples-java/wiki/OAuth">wiki</a> for details."</p>